base/strings/basic.jl

```diff
@@ -255,7 +255,7 @@ end
 
 Get the previous valid string index before `i`.
 Returns a value less than `1` at the beginning of the string.
-If `nchar` argument is given the function goes back `nchar` charsacters.
+If `nchar` argument is given the function goes back the `nchar` characters.
```

Comment:
Sorry, my suggestion about "the" was about "If the `nchar`...", not about the second occurrence (but I'm not a native speaker).